src/main/webapp/app/filters/boldMatchFilter.js

```diff
@@ -6,6 +6,6 @@ sage.filter('boldMatch', function() {
       }
     }
 
-    return "";
+    return input.replace(match, "");
```

Comment:
Should this be when input is defined? Moved in the condition block above? And still return empty string here? Or should it just return input as null or undefined?

Reply:
> Should this be when input is defined? Moved in the condition block above? And still return empty string here? Or should it just return input as null or undefined?

I need to see what happens between this.